pkg/sentry/devices/nvproxy/frontend_unsafe.go

```diff
@@ -47,6 +47,26 @@ func rmControlInvoke[Params any](fi *frontendIoctlState, ioctlParams *nvgpu.NVOS
 	return n, nil
 }
 
+func ctrlAttachGpuInvoke(fi *frontendIoctlState, ioctlParams *nvgpu.NVOS54Parameters, ctrlParams *nvgpu.NV00FD_CTRL_ATTACH_GPU_PARAMS) (uintptr, error) {
+	origDevDescriptor := ctrlParams.DevDescriptor
+	DevDescriptor, _ := fi.t.FDTable().Get(int32(origDevDescriptor))
```

Comment:
In Golang, local variable names should not start with upper case. Same for `DevDesc` below.

Reply:
Eugh damn, thanks, will fix. If there's a linter/fmter I could be running pre-push LMK.

Reply:
You can run `make test TARGETS=pkg/sentry/devices/nvproxy/...` which will run the `nogo` tests which have some linting, but I'm not sure that covers this particular problem. It's just Go style: https://google.github.io/styleguide/go/decisions.html#variable-names

Reply:
Thanks for the info. Yeh I looked in CONTRIBUTING and saw there was a style guide I could follow but no automated tool. My eye isn't good at catching these golang lints yet.